oshc/main/static/main/css/app.css

```diff
@@ -402,3 +402,21 @@ footer .container {
 .zone {
   display:none;
 }
+
+/* CONTEST SUBMISSION PAGE CSS STARTS */
+.vcenter {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  height: 90vh;
+  z-index: -100;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.subtitle {
+  margin: 0;
+}
```

Comment:
👍 

Can you please squash the commits into one, then we are good to go with this one! 🚀

Reply:
@tapasweni-pathak done 👍

Reply:
Not yet :( [![Screenshot from Gyazo](https://gyazo.com/d7f6311aa8699e6dcb7d33db9a8e26df/raw)](https://gyazo.com/d7f6311aa8699e6dcb7d33db9a8e26df). Add a commit message *Move confirmation text to the center (v,h)*.

Reply:
@tapasweni-pathak check it now 😄